data/assets/scene/digitaluniverse/constellationbounds.asset

```diff
@@ -16,7 +16,7 @@ local object = {
     Type = "RenderableConstellationBounds",
     Enabled = false,
     File = data .. "bound_20.dat",
-    ConstellationFile = data .. "constellations.dat"
+    ConstellationNamesFile = data .. "constellations.dat",
```

Comment:
Since we're renaming this property anyways, how about removing the "Constellation" part of the name? It is implied by the renderable name :)